spec/acceptance/default_parameters_jira_10_spec.rb

```diff
@@ -0,0 +1,182 @@
+# frozen_string_literal: true
+
+require 'spec_helper_acceptance'
+
+def on_supported_os
+  (fact('os.family') == 'Debian' and fact('os.release.major') == '22.04') or (fact('os.family') == 'RedHat' and fact('os.release.major') >= '8')
+end
+
+prepare = <<-EOS
+  package {'diffutils':
+    ensure  => installed
+  }
+  file_line{'enable show_diff':
+    path => '/etc/puppetlabs/puppet/puppet.conf',
+    line => 'show_diff = true'
+  }
+EOS
+
+pre = <<-EOS
+  if $facts['os']['family']  == 'RedHat' {
+    $java_package = 'java-17-openjdk'
```

Comment:
should those defaults be added to the actual puppet classes? Or at least be documented in the README.md?

Reply:
Addred a comment on that in the readme.MD